// this should cause an exception
var x = require('unknown');